### DISCUSSION TITLE
Need Help: Start WikiJS as Service

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#4301] [Help / Questions]
Hello,
I'm currently installing Wiki Js.

I am using this guide: https://docs.requarks.io/install/linux

This also works wonderfully, right down to the point: Run as service
It just doesn't work. Wiki Js ONLY works if I start this manually in the console (node ​​server) then I have to call my domain with port (www.yourdomain.com:3000) as soon as I close the console, Wiki JS no longer works, which is understandable.

As a background:
- V server
- Plesk
- NoteJS 12.4.0
- Wiki should run on a subdomain (wiki.youdomain.com)

The script I used from the guide does not work:
```
[Unit]
Description = Wiki.js
After = network.target

[Service]
Type = simple
ExecStart = / usr / bin / node server
Restart = always
# Consider creating a dedicated user for Wiki.js here:
User = nobody
Environment = NODE_ENV = production
WorkingDirectory = / var / wiki

[Install]
WantedBy = multi-user.target
```

I have adapted the WorkingDirectory to exactly where my Wiki folder is ...

Can someone help me?

**Comment by PythonCoderAS** [comment]
You **cannot** have the spaces in the paths. A space changes the path entirely. Use this config: 
```
[Unit]
Description = Wiki.js
After = network.target

[Service]
Type = simple
ExecStart = /usr/bin/node server
Restart = always
# Consider creating a dedicated user for Wiki.js here:
User = nobody
Environment = NODE_ENV = production
WorkingDirectory = /var/wiki

[Install]
WantedBy = multi-user.target
```
